napi-inl.h

```diff
@@ -3453,6 +3453,30 @@ inline const napi_node_version* VersionManagement::GetNodeVersion(Env env) {
   return result;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// JSON class
+////////////////////////////////////////////////////////////////////////////////
+inline Value JSON::Parse(Env env, String json_string) {
+  Object json = env.Global().Get("JSON").As<Object>();
+  Function parse = json.Get("parse").As<Function>();
+  return parse.Call(json, { json_string });
+}
+
+inline Value JSON::Stringify(Env env,
```

Comment:
I'm not sure but return `String` type might be better than `Value`. WDYT?
(FYI, ToString() returns `String` type)

Reply:
Yes you're right.